partner_users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class V1::PartnerUsersController < V1::BaseController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.